### ISSUE TITLE
Can't connect to Field Papers or OSM

### ISSUE THREAD

**Issue by mojodna** [posm/posm#350]
Gem `puma` is missing. Something may be missing from the EC2 ISO creation instructions. I'll build in a new environment to ensure that steps are complete.

**Comment by mojodna** [comment]
Ruby is actually missing entirely.

**Comment by mojodna** [comment]
https://github.com/Cadasta/aws-ses and https://github.com/Cadasta/s3_direct_upload no longer exist (fp-web has direct dependencies on these).

@ian-ross do you happen to still have working copies of the forked versions? If I have to, I can dig out versions from older ISOs, but if there's an easier way, I'd love it.

**Comment by mojodna** [comment]
Partial installation log:

```
Successfully installed bundler-2.0.1
1 gem installed
Fetching gem metadata from https://rubygems.org/..............
Fetching https://github.com/Cadasta/aws-ses
fatal: could not read Username for 'https://github.com': No such device or address

Retrying `git clone 'https://github.com/Cadasta/aws-ses' "/opt/fp/fp-web/vendor/bundle/ruby/2.4.0/cache/bundler/git/aws-ses-35742ead9dff8ee5fed18592cbffcdc04a6646ec" --bare --no-hardlinks --quiet` due to error (2/4): Bundler::Source::Git::GitCommandError Git error: command `git clone 'https://github.com/Cadasta/aws-ses' "/opt/fp/fp-web/vendor/bundle/ruby/2.4.0/cache/bundler/git/aws-ses-35742ead9dff8ee5fed18592cbffcdc04a6646ec" --bare --no-hardlinks --quiet` in directory /opt/fp/fp-web has failed.
fatal: could not read Username for 'https://github.com': No such device or address
```

**Comment by mojodna** [comment]
posm/posm-build@9115f19 should handle the OSM end of things.

The underlying error (from `bootstrap.log`) was:

```
Bundler could not find compatible versions for gem "bundler":
  In Gemfile:
    rails (= 4.2.8) was resolved to 4.2.8, which depends on
      bundler (< 2.0, >= 1.3.0)

  Current Bundler version:
    bundler (2.0.1)
This Gemfile requires a different version of Bundler.
```
